gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/controller/GraphControllerTest.java

```diff
@@ -111,18 +107,18 @@ void getGraphs_ReturnsGraphsAsList_whenSuccessful() throws Exception {
         assertEquals(200, getGraphsResponse.getResponse().getStatus());
     }
 
-    @Test
-    void authEndpointShouldReturn200StatusAndTokenWhenValidUsernameAndPassword() throws Exception {
-        final String authRequest = "{\"username\":\"javainuse\",\"password\":\"password\"}";
-        when(authService.getToken(any(JwtRequest.class))).thenReturn("token received");
-
-        final MvcResult result = mvc.perform(post("/auth")
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .content(authRequest)).andReturn();
-
-        assertEquals(200, result.getResponse().getStatus());
-        assertEquals("token received", result.getResponse().getContentAsString());
-    }
+//    @Test
+//    void authEndpointShouldReturn200StatusAndTokenWhenValidUsernameAndPassword() throws Exception {
```

Comment:
Comments?

Reply:
No longer needed as auth is not handled inside rest